icu4c/source/i18n/unicode/calendar.h

```diff
@@ -167,6 +167,19 @@ class BasicTimeZone;
  * to ~5,800,000 CE. Programmers should use the protected constants in `Calendar` to
  * specify an extremely early or extremely late date.
  *
+ * <p>
+ * The Japanese calendar uses a combination of era name and year number.
+ * When an empror of Japan abdicates and a new emperor ascends the throne,
```

Comment:
nit: typo in emperor.
(since this is a public header)

Reply:
Right, `empror` -> `emperor` (1st instance)

Reply:
will fix this.